src/index.ts

```diff
@@ -0,0 +1,233 @@
+import p from 'path'
+import fs from 'fs'
+import type babelCore from '@babel/core'
+import type {Primitive} from 'type-fest'
+import {getReplacement, requireFromString} from './helpers'
+
+type VisitorState = {
+  file: {
+    opts: babelCore.TransformOptions
+  }
+}
+
+export default function prevalPlugin(
+  babel: typeof babelCore,
+): babelCore.PluginObj<VisitorState> {
+  const {types: t, template, transformFromAstSync} = babel
```

Comment:
I assume `t` here refers to type?   I would prefer something a bit more descriptive here personally...

Reply:
Generally I agree, but this is [one of the lines that did not have to change to be compatible with TS](https://github.com/OpenSourceRaidGuild/babel-plugin-preval/blob/main/src/index.js#L9).